src/mesh/MeshPacketQueue.cpp

```diff
@@ -124,9 +131,10 @@ bool MeshPacketQueue::replaceLowerPriorityPacket(meshtastic_MeshPacket *p)
     // Check if the packet at the back has a lower priority than the new packet
     auto &backPacket = queue.back();
```

Comment:
I suggest removing the reference here. Maybe auto *backPacket to make it clearer what the type is.
Calling pop_back while holding a reference might not always be safe.